src/main/java/bio/terra/app/configuration/ApplicationConfiguration.java

```diff
@@ -345,6 +346,15 @@ public ObjectMapper objectMapper() {
         .registerModule(new JavaTimeModule());
   }
 
+  @Bean("daoObjectMapper")
+  public ObjectMapper daoObjectMapper() {
```

Comment:
This change is also from another PR, for some reason github is confused and showing diffs from commits that have already been merged to `develop`.